utilities/helpers.sh

```diff
@@ -90,6 +90,34 @@ function wait_for_all_healthy_nodes() {
     done
 }
 
+function wait_for_raft_sync() {
+    if [ "$(curl -s -o /dev/null -w "%{http_code}" localhost:${WEAVIATE_PORT}/v1/cluster/statistics)" == "200" ]; then
+        echo_green "Wait for Weaviate Raft schema to be in sync"
+        nodes_count=$(curl -sf localhost:${WEAVIATE_PORT}/v1/nodes | jq '.nodes | length')
+        declare -A nodes
```

Comment:
It seems this is caused because I have an old bash version (older than 4.0), maybe we could add in the dependency checks that the bash version of the system is older or equal to 4.0. This is what my system shows:
```
bash --version
GNU bash, version 3.2.57(1)-release (arm64-apple-darwin23)
Copyright (C) 2007 Free Software Foundation, Inc.
```

Reply:
This is a very good finding... I need to change this part to support bash 3

Reply:
Same here. 

MacOS X, zsh 5.9 (x86_64-apple-darwin23.0)

I believe zsh is the Mac default now.

Reply:
I have removed the need for updating the bash version to newer one. Those map structs are not being used in this script anymore